app/src/main/java/fr/free/nrw/commons/nearby/NearbyMapFragment.java

```diff
@@ -256,6 +302,27 @@ public void updateMapSignificantlyForCurrentLocation() {
         }
     }
 
+    /**
+     * Initialize all views.
+     */
+    private void initViews() {
+        Timber.d("initViews called");
+        View view = ((NearbyFragment)getParentFragment()).view;
```

Comment:
View elements that are part of `NearbyFragment` should ideally be not accessed directly in `NearbyMapFragment`. 

Either try to fix it or atleast add a `TODO` to fix it in the future.

Reply:
@maskaravivek I have added a TODO as a reminder, I would like to work on it in a future PR :)

Reply:
Sure, you can pick it up. Create a separate issue for it before starting. :)

Reply:
@maskaravivek do you think we can postpone this TODO and merge this code as is? Since it will directly be included to alpha, maybe we can merge it when TODO PR is also ready.